spec/acceptance/should_manage_values_spec.rb

```diff
@@ -198,116 +198,121 @@ def getManifest(keypath, vendor_path, phase)
   end
 
   context "Start testing should_manage_values" do
+    agent_is_x64 = false
+
     windows_agents.each do |agent|
+      agent_is_x64 = is_x64(agent)
```

Comment:
With multiple nodes in the nodeset, last one wins, which doesn't seem accurate if there are both 64-bit and 32-bit nodes...